packages/datadog-plugin-jest/src/index.js

```diff
@@ -76,37 +91,60 @@ class JestPlugin extends Plugin {
           resolve()
         }
       })
+    }).catch(() => {
+      isGitUploadFailed = true
     })
 
-    this.addSub('ci:jest:test-suite:skippable', ({ onResponse, onError }) => {
+    this.addSub('ci:library:configuration', ({ onResponse, onError }) => {
```

Comment:
the information returned could be used by other testing frameworks, but you're right that it wouldn't work because of the two-way communication. I'll namespace it 👍